lib/http_clients/b2b.ex

```diff
@@ -0,0 +1,51 @@
+defmodule HttpClients.B2b do
+  @moduledoc """
+  Client for B2b calls
+  """
+  alias HttpClients.B2b.Indication
+
+  @spec create_indication(Tesla.Client.t(), String.t()) :: {:error, any} | {:ok, Tesla.Env.t()}
+  def create_indication(%Tesla.Client{} = client, indication) do
+    case Tesla.post(client, "/v2/indications", indication) do
+      {:ok, %Tesla.Env{status: 201} = response} -> {:ok, build_indication(response.body["data"])}
+      {:ok, %Tesla.Env{} = response} -> {:error, response}
+      {:error, reason} -> {:error, reason}
+    end
+  end
+
+  defp build_indication(indication) do
+    %Indication{id: indication["id"]}
+  end
+
+  @spec client(String.t(), String.t(), String.t(), String.t()) :: Tesla.Client.t()
+  def client(base_url, client_id, client_secret, api_key \\ nil) do
+    headers = authorization_headers(client_id, client_secret, api_key)
+
+    middleware = [
+      {Tesla.Middleware.BaseUrl, base_url},
+      Tesla.Middleware.JSON,
+      {Tesla.Middleware.Headers, headers},
+      {Tesla.Middleware.Retry, delay: 1_000, max_retries: 3},
+      {Tesla.Middleware.Timeout, timeout: 15_000},
+      {Tesla.Middleware.Logger, filter_headers: ["Authorization"]},
+      Goodies.Tesla.Middleware.RequestIdForwarder
+    ]
+
+    Tesla.client(middleware)
+  end
+
+  defp authorization_headers(client_id, client_secret, nil) do
+    header = [
```

Comment:
```suggestion
     [
```
alí na linha 45 tbm dai